app/services/user.service.js

```diff
@@ -221,8 +221,11 @@ module.exports = {
    * @param newPhoneNumber
    * @returns {Promise}
    */
-  updatePhoneNumber: (externalId, newPhoneNumber) => {
-    return getAdminUsersClient().updatePhoneNumberForUser(externalId, newPhoneNumber)
-      .catch(error => new Error(`There has been a problem updating the phone number: ${error}`))
+  updatePhoneNumber: async function (externalId, newPhoneNumber) {
+    try {
+      return await getAdminUsersClient().updatePhoneNumberForUser(externalId, newPhoneNumber)
+    } catch (err) {
+      throw new Error('There has been a problem updating the phone number. Please try again')
```

Comment:
👍  removing inlining the unknown thrown error

👍  reads clearer 

If this method doesn't actually do anything with the error it feels like this would be more appropriate either: 
* higher up the stack, responding to the request with this message (potentially context dependant) 
* lower down the stack, wrapping the lower level HTTP call to admin users with some kind of formatted error